### ISSUE TITLE
How does VCFComparator file variants

### ISSUE THREAD

**Issue by patidarr** [HuntsmanCancerInstitute/USeq#6]
Hi,

When I ran intersectBed on a vcf file, all 53 variants in the vcf files are inside the bed region, but when I ran VCFComparator with the same vcf file with 53 variants it it only reads 49 from the file:

79713738        Interrogated bps in key
79713738        Interrogated bps in test
79713738        Interrogated bps in common
49      Key variants
49      Key variants in shared regions

curious where does the remaining 4 variants go?

Thanks,
Rajesh

**Comment by DavidAustinNix** [comment]
Hello Rajesh,

If you would, post your send me your vcf and bed files and I’ll test them.

Alternatively, set the -p option and look at the intersecting and non intersecting vcf files to find out where those 4 variants went.

-cheers, David



--
David Austin Nix, PhD
Director, Bioinformatics Shared Resource
Huntsman Cancer Institute (HCI-South 1704)
University of Utah
W (801) 587-4611
C (801) 901-0649
david.nix@hci.utah.edu<mailto:david.nix@hci.utah.edu>
http://huntsmancancer.org/bioinformatics



On Jan 7, 2019, at 8:29 AM, Rajesh Patidar <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi,

When I ran intersectBed on a vcf file, all 53 variants in the vcf files are inside the bed region, but when I ran VCFComparator with the same vcf file with 53 variants it it only reads 49 from the file:

79713738 Interrogated bps in key
79713738 Interrogated bps in test
79713738 Interrogated bps in common
49 Key variants
49 Key variants in shared regions

curious where does the remaining 4 variants go?

Thanks,
Rajesh
